index.html

```diff
@@ -400,12 +399,14 @@ <h3>Skills</h3>
                     <div class="user-details">
                         
                      <div class="form-list">
-                        <input type="text" placeholder="First Name" maxlength="30" required>
+                        <label for="first-name">First Name</label>
+                        <input type="text" id="first-name" placeholder="First Name" maxlength="30" required>
+                        <label for="last-name">Last Name</label>
                         <input type="text" placeholder="Last Name" maxlength="30" id="last-name" required>
                      </div>
-                           
+                                <label for="email-id">Email address</label>
                                 <input type="email" placeholder="Email address" size="30" id="email-id" required >
-        
+                                <label for="message">Write something</label>
                                 <textarea name="Message" id="message" placeholder="Write me something..." id="text-box" type="text"  maxlength="500" required></textarea>
                     
                             <div class="button">
```

Comment:
Kindly check your buttons and ensure the interactions are implemented as stated in the design😉 currently, the interaction for "pressed" or "active" has not been implemented🥲👍 Kindly see the interaction states from the design and make the necessary change👍

<img width="293" alt="button interactions" src="https://user-images.githubusercontent.com/64643995/229762028-b9d6749a-3d96-4b3b-b50f-8e1718eaf23a.png">